socialstream/chat/chat.py

```diff
@@ -159,242 +225,334 @@ def get_env_agents(env_agent_combo) -> None:
     return env, agents, environment_messages
 
 
-POSITION_CHOICES = ["First Agent", "Second Agent"]
+MODEL_LIST = [
+    "gpt-4o-mini",
+    "gpt-4o",
+    "gpt-4",
+    "together_ai/meta-llama/Llama-3-70b-chat-hf",
+    "together_ai/meta-llama/Llama-3-8b-chat-hf",
+    "together_ai/mistralai/Mixtral-8x22B-Instruct-v0.1",
+]
 
 
 def chat_demo() -> None:
-    """
-    State machine: HUMAN_WAITING (waiting for human input) -> HUMAN_SPEAKING -> MODEL_WAITING == MODEL_SPEAKING (model generate) -> HUMAN_WAITING
-    Use "active" to indicate whether the conversation is still ongoing
-    Adapted from sotopia.server.arun_one_episode
+    initialize_session_state()
 
-    # TODO manually specify the agent goals
-    """
+    with st.expander("Create your scenario!", expanded=True):
+        scenarios = st.session_state.env_mapping
+        agent_list_1, agent_list_2 = st.session_state.agent_mapping
+
+        scenario_col, model_col = st.columns(2)
+        with scenario_col:
+            scenario_choice = st.selectbox(
+                "Choose a scenario:",
+                scenarios.keys(),
+                disabled=st.session_state.active,
+                index=0,
+            )
+        with model_col:
+            model_choice = st.selectbox(
+                "Choose a model:", MODEL_LIST, disabled=st.session_state.active, index=0
+            )
 
-    initialize_session_state()
+        agent_col1, agent_col2 = st.columns(2)
+        with agent_col1:
+            agent_choice_1 = st.selectbox(
+                "Choose the first agent:",
+                agent_list_1.keys(),
+                disabled=st.session_state.active,
+                index=0,
+            )
+        with agent_col2:
+            agent_choice_2 = st.selectbox(
+                "Choose the second agent:",
+                agent_list_2.keys(),
+                disabled=st.session_state.active,
+                index=1,
+            )
+        print(agent_choice_1, agent_choice_2)
+        user_position = st.selectbox(
```

Comment:
Use Agent 1/2 or Agent 1/2's name as content?

Reply:
We are not using this script anymore right?

Reply:
That is chat_new.py as I rename it